torchgeo/trainers/utils.py

```diff
@@ -89,7 +91,6 @@ def load_state_dict(model: Module, state_dict: "OrderedDict[str, Tensor]") -> Mo
         del state_dict["fc.weight"], state_dict["fc.bias"]
```

Comment:
Should we say `if expected_num_classes and num_classes != expected_num_classes` to avoid warning when there is no layer?

Reply:
yep